.github/workflows/build-test.yml

```diff
@@ -73,6 +73,10 @@ jobs:
           cargo fmt --version
           cargo clippy --version
 
+      - name: Initialize submodules
```

Comment:
Please update the checkout action instead:

```
      - uses: actions/checkout@v3
        with:
          submodules: 'true'
```

Reply:
Actually, instead of that, can you just checkout the DPE submodule only? It's important to verify that this workflow passes when the RTL submodule is not populated.